tests/libTestUtils/TestUtils.cpp

```diff
@@ -209,18 +229,48 @@ DequeOfShard GenerateDequeueOfShard(size_t size) {
 
 CoSignatures GenerateRandomCoSignatures() { return CoSignatures(Dist1to99()); }
 
+// unsigned char randchar() {
```

Comment:
are these commented out codes still useful, otherwise plz remove

Reply:
Fixed in incomming commit.